### ISSUE TITLE
Event over multiple days possibility

### ISSUE THREAD

**Issue by peec** [alphasights/ember-calendar#27]
Great addon!

Are there any future plans to support event lasting for multiple days?

**Comment by Arkham** [comment]
Hey @peec, do you mean an event spanning over multiple full days (i.e. starting on monday and ending onthursday) or an event starting at one hour of a day and ending another hour of a different day? (i.e. starting today at 9:30 and ending tomorrow 12:30).

We thought about adding those events, but haven't started working on them actively. I would love to hear your thoughts on the matter though :)

**Comment by peec** [comment]
Yes, I mean multiple full days so the calendar can be used for e.g vacation planning to show when people are gone from work (my usecase).

Though a  feature with specific hours would be even better I guess,

Google has this kind of feature, in Google Calendar to my knowledge you have to click edit on the calendar item and a new screen opens where you can select from dates / to dates with timestamps.

So I guess a similar thing could be done on ember-calendar, where you have a screen for each / modal like where you can select from date and to date
